src/lib.rs

```diff
@@ -141,6 +144,28 @@ impl FixedBitSet
         }
     }
 
+    pub fn count_ones<T: IndexRange>(&self, range: T) -> usize
+    {
+        let start = range.start().unwrap_or(0);
+        let end = range.end().unwrap_or(self.length - 1);
```

Comment:
It should be a half open range (start inclusive, end exclusive), this way it's consistent with how slicing a vec works for example.